src/main/java/com/microtripit/mandrillapp/lutung/controller/MandrillExportsApi.java

```diff
@@ -17,11 +18,16 @@
  *
  */
 public class MandrillExportsApi {
-	private static final String rootUrl = MandrillUtil.rootUrl;
 	private final String key;
+	private final String rootUrl;
+
+	public MandrillExportsApi(final String key, final String url) {
```

Comment:
I wanted to modify the original constructor but since this is a public class with a public constructor that would break backwards compatibility which is why I chose overloading.